dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/WSWidgetsUtils.java

```diff
@@ -225,9 +225,13 @@
               .onErrorResumeNext(throwable -> Observable.empty())
               .map(myStore -> wsWidget);
         case APP_META:
-          return GetAppMetaRequest.ofAction(url, bodyInterceptor, httpClient, converterFactory,
-              tokenInvalidator, sharedPreferences, appBundlesVisibilityManager)
-              .observe(bypassCache, bypassServerCache)
+        case NEW_APP:
+          return Observable.zip(
```

Comment:
App_meta was not depending on the bonus call. It is not heavy, anyway, but just commenting in case it was a mistake